python-sdk/example_dags/example_load_file_redshift.py

```diff
@@ -0,0 +1,70 @@
+import os
+from datetime import datetime
+
+from airflow.models import DAG
+
+from astro import sql as aql
+from astro.constants import FileType
+from astro.files import File
+from astro.table import Metadata, Table
+
+# To create IAM role with needed permissions,
+# refer: https://www.dataliftoff.com/iam-roles-for-loading-data-from-s3-into-redshift/
+REDSHIFT_NATIVE_LOAD_IAM_ROLE_ARN = os.getenv("REDSHIFT_NATIVE_LOAD_IAM_ROLE_ARN")
+AWS_CONN_ID = "aws_conn"
+
+default_args = {
+    "owner": "airflow",
+    "retries": 1,
+    "retry_delay": 0,
+}
+
+dag = DAG(
+    dag_id="example_load_file_redshift",
+    start_date=datetime(2019, 1, 1),
+    max_active_runs=1,
+    schedule_interval=None,
+    default_args=default_args,
+    catchup=False,
+)
+
+
+with dag:
+    # [START load_file_redshift_example_1]
```

Comment:
@rajaths010494 no changes in documents?

Reply:
Yes, I checked for the previous place where the redshift `load_file_example` was used but couldn't find it anywhere in docs.